workspaces/liferay-partner-workspace/client-extensions/liferay-partner-etc-spring-boot/src/main/java/com/liferay/partner/QueueListener.java

```diff
@@ -95,20 +91,55 @@ public void listener(
 				return;
 			}
 
-			if (_log.isInfoEnabled()) {
-				_log.info(
-					StringBundler.concat(
-						"Updating account ", salesforceAccountKey,
-						" with name ",
-						koroneikiAccountJSONObject.getString("name")));
+			if (routingKey.equals("koroneiki.account.update")) {
```

Comment:
Can you sort these if statements? also they should be else ifs cause a message would only enter one of these if blocks